plugins/nginx-vhosts/templates/nginx.conf.sigil

```diff
@@ -12,7 +12,7 @@ server {
   access_log  /var/log/nginx/{{ $.APP }}-access.log;
   error_log   /var/log/nginx/{{ $.APP }}-error.log;
 {{ if (and (eq $listen_port "80") ($.SSL_INUSE)) }}
-  return 301 https://$host:{{ $.NGINX_SSL_PORT }}$request_uri;
+  return 301 https://$host:{{ $.PROXY_SSL_PORT }}$request_uri;
```

Comment:
This value is set [here](https://github.com/OmgImAlexis/dokku/blob/master/plugins/nginx-vhosts/functions#L298) so you also need to modify that...